Week-3/exercise_3_3/index.js

```diff
@@ -0,0 +1,21 @@
+function createIncrement() {
+  let count = 0;
+  function increment() {
+    count++;
+  }
+  let message = `Count is ${count}`;
+  function log() {
+    console.log(message);
+  }
+  return [increment, log];
+}
+
+const [increment, log] = createIncrement();
+
+increment();
+increment();
+increment();
+log();
+
+// What is logged?
+// It will log: Count is 0
```

Comment:
Add explanation why it will be 0 and also add new line

Reply:
Because message is already defined on initizalition  let message = `Count is ${count}`;